pkg/server/api/controllers/workspace/workspace.go

```diff
@@ -61,6 +62,8 @@ func GetWorkspace(ctx *gin.Context) {
 //	@id				ListWorkspaces
 func ListWorkspaces(ctx *gin.Context) {
 	workspaces, err := db.ListWorkspaces()
+	verbose := ctx.Query("verbose")
```

Comment:
I suggest you take a look at [swag](https://github.com/swaggo/swag/blob/master/README.md). That is a package that we use to generate api docs from comments placed above methods.

You'll notice that you can define query param annotations as well.

E.g. in their README:
```go
// @Param        q    query     string  false  "name search by q"  Format(email)
```

After that, generate the client with `./hack/swagger.sh`

Reply:
completed